lib/fileutils.rb

```diff
@@ -146,11 +168,14 @@ def cd(dir, verbose: nil, &block) # :yield: dir
   module_function :chdir
 
   #
-  # Returns true if +new+ is newer than all +old_list+.
-  # Non-existent files are older than any file.
+  # Returns +true+ if the file at path +new+
+  # is newer than all the files at paths in array +old_list+.;
```

Comment:
I think the period is a typo

```suggestion
  # is newer than all the files at paths in array +old_list+;
```

Reply:
Fixed.